From: Michael Froman <mfroman@mozilla.com>
Date: Mon, 4 Dec 2023 12:57:00 -0600
Subject: Bug 1867099 - (fix-66b7275561) disable wgc capture yellow-line
 removal. r?ng!

This code won't build until we support building with
Win 10 SDK v10.0.20348.0 or newer.
Mercurial Revision: https://hg.mozilla.org/mozilla-central/rev/0aac94794aad2ddb637f5076bc08706a11866737
---
 modules/desktop_capture/win/wgc_capture_session.cc | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/modules/desktop_capture/win/wgc_capture_session.cc b/modules/desktop_capture/win/wgc_capture_session.cc
index 8c74c2bf24..86afc52411 100644
--- a/modules/desktop_capture/win/wgc_capture_session.cc
+++ b/modules/desktop_capture/win/wgc_capture_session.cc
@@ -188,6 +188,11 @@ HRESULT WgcCaptureSession::StartCapture(const DesktopCaptureOptions& options) {
     }
   }
 
+// Until Mozilla builds with Win 10 SDK v10.0.20348.0 or newer, this
+// code will not build.  Once we support the newer SDK, Bug 1868198
+// exists to decide if we ever want to use this code since it is
+// removing an indicator that capture is happening.
+#if !defined(WEBRTC_MOZILLA_BUILD)
   // By default, the WGC capture API adds a yellow border around the captured
   // window or display to indicate that a capture is in progress. The section
   // below is an attempt to remove this yellow border to make the capture
@@ -199,6 +204,7 @@ HRESULT WgcCaptureSession::StartCapture(const DesktopCaptureOptions& options) {
           &session3))) {
     session3->put_IsBorderRequired(false);
   }
+#endif
 
   allow_zero_hertz_ = options.allow_wgc_zero_hertz();
 
